packages/typespec-ts/src/modular/helpers/operationHelpers.ts

```diff
@@ -143,17 +85,18 @@ export function getSendPrivateFunction(
 
 export function getDeserializePrivateFunction(
   operation: Operation,
-  needSubClient: boolean,
-  needUnexpectedHelper: boolean,
+  _needSubClient: boolean,
```

Comment:
I think we should be able to remove this safely

Reply:
Removed